torch.Size([10, 4, 2000])
INFO: Using device cuda
torch.Size([32, 4, 2000])
torch.Size([32, 4, 2000])
test
test
/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/plotly/matplotlylib/mpltools.py:368: MatplotlibDeprecationWarning:


The is_frame_like function was deprecated in Matplotlib 3.1 and will be removed in 3.3.

Traceback (most recent call last):
  File "train.py", line 124, in <module>
    train(net=net, device=device, batch_size=wandb.config.batch_size, lr=wandb.config.lr, epochs=wandb.config.epochs)
  File "train.py", line 49, in train
    wandb.log({"predict": predict_plot, "ground truth": ground_truth_plot})
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/__init__.py", line 746, in log
    run.log(row, commit, step, sync, *args, **kwargs)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/wandb_run.py", line 651, in log
    self.history.add(row, *args, step=step, commit=commit, **kwargs)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/history.py", line 139, in add
    self._write()
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/history.py", line 242, in _write
    self._transform()
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/history.py", line 225, in _transform
    self.row = data_types.history_dict_to_json(self._run, self.row)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/data_types.py", line 1618, in history_dict_to_json
    payload[key] = val_to_json(run, key, val, step=step)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/data_types.py", line 1648, in val_to_json
    val = Plotly.make_plot_media(val)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/data_types.py", line 1196, in make_plot_media
    val = util.ensure_matplotlib_figure(val)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/util.py", line 290, in ensure_matplotlib_figure
    "You attempted to log an empty plot, pass a figure directly or ensure the global plot isn't closed.")
ValueError: You attempted to log an empty plot, pass a figure directly or ensure the global plot isn't closed.
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/history.py", line 242, in _write
    self._transform()
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/history.py", line 225, in _transform
    self.row = data_types.history_dict_to_json(self._run, self.row)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/data_types.py", line 1618, in history_dict_to_json
    payload[key] = val_to_json(run, key, val, step=step)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/data_types.py", line 1648, in val_to_json
    val = Plotly.make_plot_media(val)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/data_types.py", line 1196, in make_plot_media
    val = util.ensure_matplotlib_figure(val)
  File "/home/Wr1t3R/.local/share/virtualenvs/unet-YAYnS212/lib/python3.6/site-packages/wandb/util.py", line 290, in ensure_matplotlib_figure
    "You attempted to log an empty plot, pass a figure directly or ensure the global plot isn't closed.")
ValueError: You attempted to log an empty plot, pass a figure directly or ensure the global plot isn't closed.
